apps/etterlatte-trygdetid/src/test/kotlin/no/nav/etterlatte/trygdetid/TrygdetidServiceTest.kt

```diff
@@ -286,6 +354,43 @@ internal class TrygdetidServiceTest {
         }
     }
 
+    @Test
+    fun `skal feile ved foerstegangsbehandling uten avdoede`() {
+        val trygdetid =
+            trygdetid(
+                behandlingId = randomUUID(),
+                sakId = randomSakId(),
+                ident = "UKJENT_AVDOED",
+            )
+        val behandlingId = randomUUID()
+        val sakId = trygdetid.sakId
+        val behandling =
+            mockk<DetaljertBehandling>().apply {
+                every { id } returns behandlingId
+                every { sak } returns sakId
+                every { behandlingType } returns BehandlingType.FØRSTEGANGSBEHANDLING
+            }
+        val grunnlagUtenAvdoede = grunnlagUtenAvdoede()
+        coEvery { grunnlagKlient.hentGrunnlag(behandlingId, any()) } returns grunnlagUtenAvdoede
+        every { repository.hentTrygdetiderForBehandling(behandlingId) } returns listOf(trygdetid)
```

Comment:
Jeg lurer på om en test for førstegangsbehandling uten avdøde der vi ikke har noen trygdetid før kunne vært greit -- men det er kanskje dekt fra før

Reply:
At det skal feile på samme måte uten trygdetider, tenker du?